src/null-safety/index.md

```diff
@@ -61,33 +74,15 @@ Dart null safety support is based on the following three core design principles:
   fewer bugs, but smaller binaries and faster execution.
 
 
-## Enabling null safety {#enable-null-safety}
-
-Sound null safety is available in Dart 2.12 and Flutter 2.
-
+## Enabling/disabling null safety {#enable-null-safety}
 
-### Migrating an existing package or app {#migrate}
-
-For instructions on how to migrate your code to null safety,
-see the [migration guide][].
+Sound null safety is available in Dart 2.12 or later,
+and in Flutter 2.0 and later. Dart 3 and later will
+[_only_ support sound null safety][Dart 3 sound null safety tracking issue].
 
-{{site.alert.version-note}}
-  Before Dart 2.13, the templates used by the [`dart create`][] command
-  and IDEs aren't null safe, so you need to migrate the code they create.
-  For example:
-
-  ```terminal
-  $ dart create -t console my_cli
-  $ cd my_cli
-  $ dart migrate --apply-changes
-  ```
-{{site.alert.end}}
-
-### Behind the scenes: SDK constraints {#constraints}
-
-To make Dart treat your code as null safe,
-the [SDK constraints](/tools/pub/pubspec#sdk-constraints)
-must require a [language version][] that has null safety support.
+Sound null safety is currently enabled by setting the
```

Comment:
Since the header is removed, can we add a link target for anywhere currently referencing it? 

```suggestion
<a id="constraints"></a>
Sound null safety is currently enabled by setting the
```